app/src/main/java/org/schulcloud/mobile/data/model/CurrentUser.java

```diff
@@ -15,6 +24,8 @@
     public String schoolId;
     public String displayName;
 
+    public RealmList<RealmString> permissions;
+
     public String get_id() {
```

Comment:
I know it is not part of your PR, but I don't like the method naming here. Why not using camel case? And why naming this attribute `_id` instead of `id`? I know it is common when working with Android's SQLight API, but with Realm? /cc @pinussilvestrus